Modules/SubModule_Import/xASL_imp_Initialize.m

```diff
@@ -52,6 +52,38 @@
     imPar.BidsRoot = fullfile(imPar.BidsRoot,imPar.studyID,'rawdata');
 
     %% 4. Specify the tokens
+    
+    % Specify the imPar struct
+    imPar = xASL_imp_InitializeBuildImPar(imPar);
+    
+    % Warn the user if token aliases are invalid according to our pipeline/BIDS
+    xASL_imp_InitializeCheckTokens(imPar);
+
+    %% 5. Specify the additional details of the conversion
+    if ~isfield(imPar,'bVerbose') || isempty(imPar.bVerbose)
+        imPar.bVerbose = true;
+    end
+    if ~isfield(imPar,'bOverwrite') || isempty(imPar.bOverwrite)
+        imPar.bOverwrite  = false; % NB, the summary file will be recreated anyway and dicom conversion in temp is always done, even if dest. exists
+    end
+    if ~isfield(imPar,'visitNames') || isempty(imPar.visitNames)
```

Comment:
Why do we define these here, these are not needed for `imPar` (dcm2nii), these are needed for BIDS2legacy, right?

Reply:
That has been there forever and was just moved here...

Reply:
Yes - we can make an issue for reconstructing this function if needed...